utils/webpack.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const BB = require("bluebird");
+const webpack = BB.promisify(require("webpack"));
+const fse = require('fs-extra');
+
+module.exports = {
+  pack: async function(serverless, functionObject) {
+    serverless.cli.log(`bundling: ${functionObject.script}`);
+
+    let outputPath = path.join(serverless.config.servicePath, functionObject.script);
+
+    let config = {
+      entry: {
+        out: outputPath
+      },
+      output: {
+        filename: `${functionObject.script}.js`,
+        path: path.join(serverless.config.servicePath, 'dist'),
+      },
+      devtool: 'cheap-module-source-map',
+      target: 'web'
```

Comment:
Targeting `web` and `webworker` blindly also leads to issues with dependencies that rely on libraries like `fs`, `net`, or `tls`. When developing workers with webpack I've needed to add this to my config file:

`  node: {
    fs: "empty",
    tls: "empty",
    net: "empty",
    child_process: "empty"
  },`

Other people say the solution is to target `node` instead

Resources for research: https://github.com/webpack-contrib/css-loader/issues/447
https://webpack.js.org/configuration/node/

I'm still not 100% on what the pattern should be for CF workers + webpack

Reply:
Thanks. I've been working on a different PR which handles things a little better. I haven't needed to add anything in order to get it working though. A simple config works for me. I should have a PR up this week.

Reply:
I just put up another PR: https://github.com/cloudflare/serverless-cloudflare-workers/pull/4
I still need to fix the generator (which I found didn't produce deployable config anyway because of a regex mismatch)